examples/notebooks/vanilla-gan-tutorial.ipynb

```diff
@@ -0,0 +1,1000 @@
+{
```

Comment:
<p>you can use <code>torchvision.utils.make_grid</code></p><p>(you can left everything as it is, it was just fyi)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-team/catalyst/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>